lib/MCT.pm

```diff
@@ -111,6 +81,23 @@ sub _setup_secrets {
   $app->secrets($secrets);
 }
 
+sub _will_remove_this_once_prod_is_up_to_date {
```

Comment:
The reason for this sub is that @tempire should not do any manual changes when he reloads hypnotoad: I don't think "identifier" is correct in the production database, so that's what this method fix.